packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/bin/server/controllers/users_controller.dart

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2023, Prime Holding JSC
```

Comment:
If the newly added classes are onboarding-specific, please move it to the featature_onboarding brick.